addrs.go

```diff
@@ -64,10 +86,17 @@ func ParseWebsocketNetAddr(a net.Addr) (ma.Multiaddr, error) {
 }
 
 func parseMultiaddr(a ma.Multiaddr) (string, error) {
-	_, host, err := manet.DialArgs(a)
+	p := a.Protocols()
```

Comment:
Could you explain why this change? Why not use `DialArgs` and then [`net.SplitHostPort`](https://golang.org/pkg/net/#SplitHostPort) to get the host and the port separately? You can then prepend `ws` or `wss` based on the presence of the protocol.

Reply:
I just copied this over from #46.

`DialArgs` returns the following error: 

```
/dns/nyc-1.bootstrap.libp2p.io/tcp/443/wss is not a 'thin waist' address
```

Do you know how to address that?

Reply:
https://github.com/multiformats/go-multiaddr-net/pull/63

Reply:
Merged and released: https://github.com/multiformats/go-multiaddr-net/releases/tag/v0.1.2. @albrow, mind updating and testing?

Reply:
Thanks. @raulk @Stebalien. Just updated and the tests pass.